x11-themes/kvantum/kvantum-1.1.5.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 # taken from Kvantum/style/CMakeLists.txt
-QTMIN="6.2.0"
+QTMIN="6.6.0"
```

Comment:
The code also comes with a lot of 
> #if (QT_VERSION >= QT_VERSION_CHECK(6,8,0))
> #if (QT_VERSION >= QT_VERSION_CHECK(6,8,2))

Since we're not having a slot operator on dev-qt/qtbase (there is none needed per its subslot metadata description), but `6.8.2-r2` is our latest available version in ::gentoo, raising `QTMIN=6.8.2` would cost nothing and ensure late upgraders can't build kvantum against an old version of Qt (lacking ifdef workarounds for 6.8.2 or applying wrong ones for older versions).

Reply:
Seems we missed another thing - `Kvantum/style/CMakeLists.txt` has defined: `set(QT5_MINIMUM_VERSION "5.15.0")`.

Two options:
- Raise the ebuild minimum deps to 5.15.0
- Cleanup the min reqs as even 5.15.0 is ancient by now, and previous version 5.14.2 was dropped in 2020, so well ahead of our usual 2 year upgrade "guarantee".

Reply:
The minimum version for the qt5 deps is already higher as the the upstream defined, but cleanup is a good idea at this point.